### ISSUE TITLE
Enviroment in config.yml

### ISSUE THREAD

**Issue by archufede** [egeloen/IvoryCKEditorBundle#294]
Hi.

Is there any way to set env.isCompatible in the config.yml of symfony?
I need to enable the ckeditor for mobile in my system, and the only way i find to do it was to modify the ckeditor.js file. But that way i will have to do this change everytime i update the bundle.

**Comment by egeloen** [comment]
@archufede The env is not supported right now... Since it is supported, instead of relying on the CKEditor version shipped with the bundle, you can rely on your own version to not have to fix it after each deploy: http://symfony.com/doc/master/bundles/IvoryCKEditorBundle/usage/version.html

**Comment by archufede** [comment]
Thanks. I will use that.

Are you planning to add env configuration in the future?

**Comment by egeloen** [comment]
Maybe if I found some time but honestly it has a really low priority on my TODO list...

**Comment by egeloen** [comment]
@archufede I'm not sure I will add support for environment, let's see if other people also request it. Anyway, I just found a proper solution to your issue. The doc was incomplete and I just fixed it few days ago. Take a look to http://symfony.com/doc/current/bundles/IvoryCKEditorBundle/usage/append-javascript.html which basically explains you how to easily add append extra javascript when rendering the widget.
